terraform/environment/terraform.tfvars.json

```diff
@@ -20,7 +20,7 @@
       "sirius_api_gateway_endpoint": "https://integration.dev.lpa.api.opg.service.justice.gov.uk/v1/lpa-online-tool/lpas/",
       "sirius_api_gateway_arn": "arn:aws:execute-api:eu-west-1:288342028542:*/*/GET/lpa-online-tool/*",
       "auth_token_ttl_secs": 4500,
-      "backup_retention_period": 0,
+      "backup_retention_period": 1,
```

Comment:
What's the reason for increasing this value?

Reply:
this was for RDS to allow us to test the snapshot process. we can remove this now to zero for the time being.